crates/recursor/src/lib.rs

```diff
@@ -132,6 +135,21 @@ fn maybe_strip_dnssec_records(query_has_dnssec_ok: bool, lookup: Lookup, query:
     Lookup::new_with_deadline(query, records, lookup.valid_until())
 }
 
+/// Add discovered IP addresses to a NameServerConfigGroup.  This function is used by ns_pool_for_zone
+/// to build the ConfigGroup for a NameServerPool.
+fn add_config_group_recs(ips: Vec<IpAddr>, config_group: &mut NameServerConfigGroup) {
```

Comment:
Why not make this a method on `NameServerConfigGroup`?

Reply:
I moved this to NameServerConfigGroup::append_ips